index.rst

```diff
@@ -6,24 +6,219 @@
+ 
+.. figure:: /_static/m1m3004_hp_timeline_El_90.png
+   
+   Figure 1. Transition of the measured forces on each hard point when the TMA is at el=90deg. 
+
+
+In Figure 2, there are the change of the measured force for each phase/status in the hard point breakaway test, moving Negative, testing positive, and testing negative, respectively. 
+The stiffness of each curves are fitted with +-10 points from :math:`\Delta`\displacement = 0 :math:`{\mu}m`. 
+All stiffness slopes are shallower than specification (100N/:math:`{\mu}m`). 
+ 
+.. figure:: /_static/Force_displacement_salidx_100061_El_90.png
+   :scale: 45 %
+
+   Figure 2. :math:`\Delta`\Displacement versus Measured Forces for each phase during the hard point breakaway test 
+
+
+In order to check residual bumps during the movements of hard points, we adopted the error function :eq:`error_function` to fit the measured forces with respect to :math:`\Delta`\displacement for active phases when the hard points are moving toward negative and positive directions.  
+As hard points breakaway limits for each direction are different, the functions at the positive and negative in x axes were fitted separately.    
+The maxima of the bumps are about < 250\N, which correspond < 10\% of the measured forces.  
+ 
+
+.. math:: erf(x) = {\frac{2}{\sqrt{\pi}} \int_{0}^{x} e^{-t^2}\,dt}
+   :label: error_function 
+
+
+
+.. figure:: /_static/Force_displacement_fitting_residual_salidx_100061_El_90.png
+   
+   Figure 3. (Left) Same as Figure2, but fit with error function. (Right) The residual, a difference between data and error function, with respect to :math:`\Delta`\displacement 
+
+
+
+
+
+Hard Point Breakaway Test at El=0
+===================================
+
+These are results from hard point breakaway test when the TMA was positioned at el=0 deg, az=-29.69 deg. 
+In Figure 4, hard point 2 and hard point 5 were not moving to the positive direction. 
+Hardpoint 1 and hard point 6 were both staying on the position for testing positive for a shorter period of time whereas hard point 3 and hard point 4 were staying on testing negative position for a shorter period time.  
+This is because depending on the position of each hard point.  
+
+.. figure:: /_static/m1m3004_hp_timeline_El_0.png
+   
+   Figure 4. Same as Figure 1 but the TMA at el=0 deg. 
+
+The stiffness of each curves are fitted from :math:`\Delta`\displacement = 0 :math:`{\mu}m` (Figure 5). 
+
+.. figure:: /_static/Force_displacement_salidx_100056_El_0.png
+   :scale: 45 %
+
+   Figure 5. Same as Figure 2 but the TMA at el=0 deg. 
+
+
+.. figure:: /_static/Force_displacement_fitting_residual_salidx_100056_El_0.png
+   
+   Figure 6. Same as Figure 3 but the TMA at el=0 deg.   
+
+
+Hard Point Breakaway Test at el=40
+===================================
+
+.. figure:: /_static/m1m3004_hp_timeline_El_40.png
+   
+   Figure 7. Same as Figure 1 but the TMA at el=40 deg. 
```

Comment:
I know it can be a bit repetitive, but imagine someone reading this document and having to scroll up and down every time to remember what was in the description of the plot. I suggest you repeat the explanation on each of them.